test/TestAppUtils/TestFrame.cs

```diff
@@ -128,6 +136,22 @@ protected override void OnApplyTemplate()
             _unhandledExceptionReportingTextBox = (TextBox)GetTemplateChild("UnhandledExceptionReportingTextBox");
         }
 
+        private void _innerFrameInLabDimensions_Click(object sender, RoutedEventArgs e)
+        {
+            if(double.IsInfinity(_pagePresenter.MaxWidth))
+            {
+                // Not CI mode, so enter it now
+                _pagePresenter.MaxWidth = 1024;
```

Comment:
The tests don't actually use this button right? Also, I'm trying to figure out why the tests are failing in the lab and I"m noticing that my local run includes I think 1 extra pixel than the screenshot from the lab shows. Is this number possibly off by one?

Reply:
Yes the tests don't actually use that button, it is only designer for developers to check if there are possible problems with the test page size. It could be that the number is of by one, measuring the screenshots is not the most precise method.

Could you tell me what number of pixels it should be instead?